util/topgen/templates/chiplevel.sv.tpl

```diff
@@ -490,6 +490,8 @@ module chip_${top["name"]}_${target["name"]} (
 
   // otp alert
   ast_pkg::ast_dif_t otp_alert;
+  assign otp_alert.p = 1'b0;
```

Comment:
o is this because nuvoton prefers we keep the ports on ast?

Reply:
yes (for now). they may decide to remove it entirely, but until then I just leave the AST untouched.